Source/Core/Core/HW/WiimoteEmu/Speaker.cpp

```diff
@@ -68,39 +68,40 @@ void stopdamnwav()
 }
 #endif
 
-void Wiimote::SpeakerData(const wm_speaker_data* sd)
+void Wiimote::SpeakerLogic::SpeakerData(const u8* data, int length)
```

Comment:
Thoughts on using a `vector` (or some other container that has length built in) instead of two parameters?

Reply:
The data comes directly from core. I could copy it to a vector first but I think that's unnecessary?

Reply:
That feels unnecessary, I didn't fully follow the data flow there (which is a little difficult with just the web diff on GitHub).
Can probably disregard this one (unless it is easily possible to use the vector or whatever it ends up as all the way)